# Chapter 3 > Exercise 11

In this problem, we will investigate the t-statistic for the null hypothesis H0 : β = 0 in simple linear regression without an intercept. To begin, we generate a predictor x and a response y as follows.
> set.seed(1)<br>
x=rnorm(100)<br>
y=2*x+rnorm(100)

(a) Perform a simple linear regression of y onto x, without an intercept. Report the coefficient estimate βˆ, the standard error of this coefficient estimate, and the t-statistic and p-value associated with the null hypothesis H0 : β = 0. Comment on these results. (You can perform regression without an intercept using the command lm(y∼x+0).)<Br>
(b) Now perform a simple linear regression of x onto y with an intercept, and report the coefficient estimate, its standard error, and the corresponding t-statistic and p-values associated with the null hypothesis H0 : β = 0. Comment on these results.<br>
(c) What is the relationship between the results obtained in (a) and (b)?<br>
(d) For the regression of Y onto X without an intercept, the t- statistic for H0 : β = 0 takes the form βˆ/SE(βˆ), where βˆ is given by (3.38), and where (see book for formula)<br>
(These formulas are slightly different from those given in Sec- tions 3.1.1 and 3.1.2, since here we are performing regression without an intercept.) Show algebraically, and confirm numerically in R, that the t-statistic can be written as (see book for formula)<br>
(e) Using the results from (d), argue that the t-statistic for the regression of y onto x is the same as the t-statistic for the regression of x onto y.<br>
(f) In R, show that when regression is performed with an intercept, the t-statistic for H0 : β1 = 0 is the same for the regression of y onto x as it is for the regression of x onto y.

In [2]:
set.seed(1)
x=rnorm(100)
y=2*x+rnorm(100)

<b>(a) Perform a simple linear regression of y onto x, without an intercept. Report the coefficient estimate βˆ, the standard error of this coefficient estimate, and the t-statistic and p-value associated with the null hypothesis H0 : β = 0. Comment on these results. (You can perform regression without an intercept using the command lm(y∼x+0).)</b><br>
> <font color="green">
    Comment: Reject null hypothesis as p-value is statistically significant.
    </font>

In [14]:
lmodel=lm(y~x+0)
summary(lmodel)


Call:
lm(formula = y ~ x + 0)

Residuals:
    Min      1Q  Median      3Q     Max 
-1.9154 -0.6472 -0.1771  0.5056  2.3109 

Coefficients:
  Estimate Std. Error t value Pr(>|t|)    
x   1.9939     0.1065   18.73   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.9586 on 99 degrees of freedom
Multiple R-squared:  0.7798,	Adjusted R-squared:  0.7776 
F-statistic: 350.7 on 1 and 99 DF,  p-value: < 2.2e-16


<b>(b) Now perform a simple linear regression of x onto y with an intercept, and report the coefficient estimate, its standard error, and the corresponding t-statistic and p-values associated with the null hypothesis H0 : β = 0. Comment on these results.</b><br>
> <font color="green">
    Comment: Summary is almost same as above.
    </font>

In [15]:
lmodel = lm(y~x)
summary(lmodel)


Call:
lm(formula = y ~ x)

Residuals:
    Min      1Q  Median      3Q     Max 
-1.8768 -0.6138 -0.1395  0.5394  2.3462 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept) -0.03769    0.09699  -0.389    0.698    
x            1.99894    0.10773  18.556   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.9628 on 98 degrees of freedom
Multiple R-squared:  0.7784,	Adjusted R-squared:  0.7762 
F-statistic: 344.3 on 1 and 98 DF,  p-value: < 2.2e-16


<b>(c) What is the relationship between the results obtained in (a) and (b)?</b><br>
> <font color="green">
    Comment: No difference observed.
    </font>

<b>(d) For the regression of Y onto X without an intercept, the t- statistic for H0 : β = 0 takes the form βˆ/SE(βˆ), where βˆ is given by (3.38), and where (see book for formula)
(These formulas are slightly different from those given in Sections 3.1.1 and 3.1.2, since here we are performing regression without an intercept.) Show algebraically, and confirm numerically in R, that the t-statistic can be written as (see book for formula)</b><br>
> <font color="green">
    Comment: t_stat is matching with t-value (18.73) in without an intercept summary.
    </font>
    

In [25]:
numa = sqrt(100-1) * sum(x*y)
deno = sqrt( sum(x^2) * sum(y^2) - sum(x*y)^2 )
t_stat = numa/deno
t_stat

[1] 18.72593

<b>(e) Using the results from (d), argue that the t-statistic for the regression of y onto x is the same as the t-statistic for the regression of x onto y.</b><br>
> <font color="green">
    Comment: t-value is same as above without intercept summary.
    </font>

In [29]:
summary(lm(x~y+0))$coeff

Estimate  Std. Error t value  Pr(>|t|)    
y 0.3911145 0.02088625 18.72593 2.642197e-34

<b>(f) In R, show that when regression is performed with an intercept, the t-statistic for H0 : β1 = 0 is the same for the regression of y onto x as it is for the regression of x onto y.</b><br>
> <font color="green">
    Comment: t-value is same as above with intercept summary.
    </font>

In [30]:
summary(lm(x~y))$coeff

Estimate   Std. Error t value    Pr(>|t|)    
(Intercept) 0.03880394 0.04266144  0.9095787 3.652764e-01
y           0.38942451 0.02098690 18.5555993 7.723851e-34